Week1/countryData.js

```diff
@@ -0,0 +1,37 @@
+const checkAndCreate = 'CREATE TABLE IF NOT EXISTS';
+const insert = `INSERT INTO country 
+                (Code, Name, Continent, Region, SurfaceArea, 
+                IndepYear, Population, LifeExpectancy, GNP, GNPOld, 
+                LocalName, GovernmentForm, HeadOfState, Capital) VALUES`;
+const CREATE_COUNTRY_TABLE = `
+                ${checkAndCreate} country(
```

Comment:
I love that you are trying to have constant for duplicates.
It would have been more relevant if you could have used this variable for countryData and cityData and not duplicate it.
If you want you can try to have this checkAndCreate const in a common file so you could use it in both query.

Reply:
Thanks a lot for having time to review my homework! Really appreciate it! Yes, that's right, I'm going to move it to dbUtils file, then I'll require from that file!